services/QuillLMS/app/services/learn_worlds_integration/user_tags_request.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module LearnWorldsIntegration
+  class UserTagsRequest < UserRequest
+
+    attr_reader :external_id, :tags
+
+    def endpoint
```

Comment:
I would use one-liner here:
`def endpoint =  "#{USER_TAGS_ENDPOINT}/#{external_id}"`

Also, `def initialize` should be the first instance method in a class